pkg/fission-cli/function.go

```diff
@@ -575,6 +578,11 @@ func fnUpdate(c *cli.Context) error {
 	if len(entrypoint) > 0 {
 		function.Spec.Package.FunctionName = entrypoint
 	}
+
+	if c.IsSet("fntimeout") {
+		function.Spec.FunctionTimeout = fnTimeout
```

Comment:
Return error when a user specifies the illegal value, just like how we do in specialization timeout.
https://github.com/fission/fission/blob/e481c234ee038f8ce8c877a4ab7d3fbf8b8b5597/pkg/fission-cli/function.go#L155-L160